bitnami/elasticsearch/templates/_helpers.tpl

```diff
@@ -255,10 +284,12 @@ Create the name of the service account to use
 {{- end -}}
 
 {{/*
-Return the proper ES curator image name
+Return the proper Storage Class
+Usage:
+{{ include "elasticsearch.storageClass" (dict "global" .Values.global "local" .Values.master) }}
 */}}
-{{- define "elasticsearch.curator.image" -}}
-{{ include "common.images.image" (dict "imageRoot" .Values.curator.image "global" .Values.global) }}
+{{- define "elasticsearch.storageClass" -}}
+{{- include "common.storage.class" (dict "global" .global "persistence" .local.persistence) -}}
 {{- end -}}
```

Comment:
I don't find the `"elasticsearch.storageClass"` helper useful. I mean, I'd simply use `"common.storage.class"` instead where needed.

Reply:
Thanks! I missed that one. There were a lot of `elasticsearch.*` helpers that could be replaced with `common` helpers. This one wasn't new, but can be omitted. 👍